examples/options_template.ini

```diff
@@ -114,19 +114,24 @@
 # default is True (yes/no can also be used)
 #use_errors: yes
 
-# jac_method: Calls scipy.optimize._numdiff.approx_derivative which
+# jac_method: This sets the options type of Jacobian used. Note: currently
+#                only numerical Jacobians are implemented.
+#default 'numerical'
+#jac_method: numerical
```

Comment:
I'm not sure this gives enough scope to name Jacobians if we add lots in the future.  Could this be `scipy-finite-diff` (or something) instead of `numerical`?  Or possibly the `num_method` should be `scipy-2point`.  I feel the fact it's scipy's should be in the name somewhere, to distinguish from other 2 point or 3 point methods we may add in the future

Reply:
I have changed the naming convention slightly, let me know what you think